### PR TITLE
Tutorial] 4. 確保計算工作持續不中斷

### PR DESCRIPTION
**Pull request by ting88510**

### CODE REVIEW

File: CCS/3. Tutorials/4. 確保計算工作能持續不中斷.md [hunk 1]
```diff
@@ -1 +1,154 @@
...
+### Step 2. Enter Screen 
+```bash=
+screen
+```
+- After entering Screen, please read the description.
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_47056d9911ee362ef05bc9e6ac33febc.png)
+
+
+- Press "Return" to insert the command
```
Comment (VigaWei) [inline CCS/3. Tutorials/4. 確保計算工作能持續不中斷.md:84]: [SUG]

1. "Return" > **Return** key
2. insert > enter

File: CCS/3. Tutorials/4. 確保計算工作能持續不中斷.md [hunk 2]
```diff
@@ -1 +1,154 @@
...
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_e1b9bebfacbf19334f64d7f9c5b4cd7b.png)
+
+
+
+### Step 3. Run the program in the screen shell
```
Comment (VigaWei) [inline CCS/3. Tutorials/4. 確保計算工作能持續不中斷.md:90]: [SUG]

program > processing

File: CCS/3. Tutorials/4. 確保計算工作能持續不中斷.md [hunk 3]
```diff
@@ -1 +1,154 @@
...
+
+- The example is as follows
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_0017c6102df15b490d4492adf889470a.png)
+
+
+
+### Step 4. Detach screen shell
+
+- When the program is running, press `ctrl + a + d`, it will show detach successful message (shell ID marked in red frame). Then, the program can run continously in the background, even if SSH is disconnected.
```
Comment (VigaWei) [inline CCS/3. Tutorials/4. 確保計算工作能持續不中斷.md:100]: [SUG]

1. program*2 > processing
2. marked in red frame > marked with red frame

File: CCS/3. Tutorials/4. 確保計算工作能持續不中斷.md [hunk 4]
```diff
@@ -1 +1,154 @@
...
+
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_017ef5c98429becb339bd36d43cdc7d2.png)
+
+
+
+### Step 5. Check the status of the running program
+
+- If you want to enter that screen shell again, insert the command to operate container and check the status of the program:
```
Comment (VigaWei) [inline CCS/3. Tutorials/4. 確保計算工作能持續不中斷.md:109]: [SUG]

1. program*2 > processing
2. insert > enter

File: CCS/3. Tutorials/4. 確保計算工作能持續不中斷.md [hunk 5]
```diff
@@ -1 +1,154 @@
...
+```bash=
+screen -r [A chunk of screen shell ID]
+```
+:::info
+:book: [<ins>More Screen command</ins>](https://blog.gtwang.org/linux/screen-command-examples-to-manage-linux-terminals/)
+:::
+
+## Method 3. Using Linux nohup command
+
+### Step 1. Launch the container using SSH and run the program
```
Comment (VigaWei) [inline CCS/3. Tutorials/4. 確保計算工作能持續不中斷.md:120]: [SUG]

1. Launch > Connect to
2. program > processing

File: CCS/3. Tutorials/4. 確保計算工作能持續不中斷.md [hunk 6]
```diff
@@ -1 +1,154 @@
...
+
+- Enter the command below to run the program in the background.
```
Comment (VigaWei) [inline CCS/3. Tutorials/4. 確保計算工作能持續不中斷.md:122]: [SUG]

program > processing

File: CCS/3. Tutorials/4. 確保計算工作能持續不中斷.md [hunk 7]
```diff
@@ -1 +1,154 @@
...
+
+```bash=
+nohup [/code_path] &
+```
+:::info
+==**Need to update:exclamation:**==
+:book: Please refer to [<ins>Lanch the container using SSH</ins>](https://man.twcc.ai/s/SJlZnSOaN#%E4%BD%BF%E7%94%A8-SSH-%E7%99%BB%E5%85%A5%E9%80%A3%E7%B7%9A).
```
Comment (VigaWei) [inline CCS/3. Tutorials/4. 確保計算工作能持續不中斷.md:129]: [SUG]

Lanch > Connect to

File: CCS/3. Tutorials/4. 確保計算工作能持續不中斷.md [hunk 8]
```diff
@@ -1 +1,154 @@
...
+:::
+- The example is as follows, terminal will return the job ID (job ID = 1 marked in red frame), and the output file of the program will be displayed in the "nohup.out"
```
Comment (VigaWei) [inline CCS/3. Tutorials/4. 確保計算工作能持續不中斷.md:131]: [SUG]

1. in > with
2. program > processing

File: CCS/3. Tutorials/4. 確保計算工作能持續不中斷.md [hunk 9]
```diff
@@ -1 +1,154 @@
...
+:book: Please refer to [<ins>Lanch the container using SSH</ins>](https://man.twcc.ai/s/SJlZnSOaN#%E4%BD%BF%E7%94%A8-SSH-%E7%99%BB%E5%85%A5%E9%80%A3%E7%B7%9A).
+:::
+- The example is as follows, terminal will return the job ID (job ID = 1 marked in red frame), and the output file of the program will be displayed in the "nohup.out" 
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_d220eceddf16a2cc6c0e29a2af8b569b.png)
+
+
+### Step 2. View the output content
+
+- Enter the following command to view the latest output of the program.
```
Comment (VigaWei) [inline CCS/3. Tutorials/4. 確保計算工作能持續不中斷.md:138]: [SUG]

program > processing

**Review by VigaWei — COMMENTED** [review]
Thx! Please revise the doc according to the comments.

**Comment by ting88510** [comment]
thanks! already revised!

**Pull request merged**